Merge Design: Configuration into develop (#103)
SW709E19/Report: report/PartDesign/Configuration.tex

\section{Configuration of devices}

> As described in \autoref{cha:maintenance}, different machines require different data to be collected in order to predict maintenance.

Reviewer: ```suggestion
As described in \autoref{cha:maintenance}, different machines require different data to be collected in order to do predictive maintenance.
```

Therefore, to support predictive maintenance of a wide variety of machines, the system should allow the sensor devices to be configured to the specific machines they are attached to.

> These configurations should be remotely accessible for viewing and updating in order to make configuration management uncomplicated.

Reviewer: ```suggestion
These configurations should be remotely accessible for viewing and updating in order to make configuration management straightforward.
```

The fundamental configuration settings are sample rate and duty cycle.
The sample rate is the number of measurements taken per second and is measured in Hz, while the duty cycle defines the amount of time the sensor is sampling versus sleeping.

> These configurations are important because they determine the amount and expressiveness of the data available for analysis, and heavily impact the battery lifetime of the sensor device.

Reviewer: ```suggestion
These configurations are important because they determine the amount and expressiveness of the data available for analysis, and heavily impact the battery expectancy of the sensor device.
```

While sampling all the time at a high sample rate could result in the most accurate predictions, it would also mean replacing batteries frequently.

> Therefore, these settings should be configured to the specific machine monitored so that the sensor is sampling enough to make useful predictions, while sleeping as much as possible to preserve its battery lifetime.

Reviewer: ```suggestion
Therefore, these settings should be configured to the specific machine monitored so that the sensor is sampling enough data to make useful predictions, while sleeping as much as possible to preserve its battery.
```
